build/scripts/build-utils.ps1

```diff
@@ -148,6 +148,7 @@ function Ensure-DotnetSdk() {
         Create-Directory $cliDir
         Create-Directory $toolsDir
         $destFile = Join-Path $toolsDir "dotnet-install.ps1"
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
>[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 [](start = 8, length = 79)

Not sure what happened in this file. Should be reverted.

Reply:
This is a necessary change. The servers that we get a lot of our bootstrapping tools from recently changed to require Tls12 which is not the default. Every branch has needed to cherry-pick this fix in order to pass CI.

Reply:
I think we should just perform a merge from master into this branch. I'll queue up a PR to add it to the automerge list as well.